configs/tsconfig.base.json

```diff
@@ -2,6 +2,8 @@
   "compilerOptions": {
     "lib": ["ES2020"],
     "allowJs": true,
+    "moduleResolution": "node",
```

Comment:
why do we need here `"moduleResolution": "node"` ?

Reply:
Cause `resolveJsonModule` doesn't work without it..

Reply:
what else does it affect?